### ISSUE TITLE
[Feature request] increase stealth by deactivating profiless

### ISSUE THREAD

**Issue by Sturmkater** [GrapheneOS/os-issue-tracker#3808]
I did set up my pixel 8 with a dummy owner profile and different case specific other profiles. In the logging screen there is the option to end the session, what i think is pretty need.
To increase stealth I would love the option to deactivate multiple users at the same time as ending the session. Deactivated, user profiles want be lost, but nothing indicates there are other profiles there. The option to activate multiple users could even made require the admin password.

That would be a nice security feature to be implemented. But in the mean time, does anybody know if deactivating multiple users could as well be done with tasker?

**Comment by Sturmkater** [comment]
Sure, I love to clarify.
![1](https://github.com/user-attachments/assets/9afac8c7-7ff5-4274-8ada-8ab906a9ae4d)
If turned of here, the user profile are not touched and persist to the next activation.
![3](https://github.com/user-attachments/assets/927f762d-cba9-4209-b772-2e47a6b5151e)
...but here, the only place I know to change user, it is no longer visible that other profiles are in existens and therefore limiting the suspicion that the current (owner) profile there in is just a dummy.
![2](https://github.com/user-attachments/assets/c9dc5327-79aa-4c53-9735-cd35a468b997)
... to reactivate it could need the admin password. If forced to enter one could still enter the duress password to wipe the entire device. If this approach does not have some holes, it would be way more convenient to quickly deactivate multiple users instead of wiping users witch afterwards have to be set up again.

**Comment by keybreak** [comment]
@thestinger 
Sorry to interject, but as far as i understand that's pretty valid concern:
1. Let's say you're stopped by cops in authoritarian country, while having some opposition materials or contacts on your device.

2. You should want to be calm and open some dummy profile without leaving **any** visible trace of having more than one profile.

3. If you fail to do so - they'll open such device with your face and you'll go to jail for contents of your device.

**Comment by keybreak** [comment]
I've meant more like street cops sweep checks, not full on Cellebrite type of situation.
Well valid or not...that's a big problem., hopefully some ideas will arise in the future. :+1:

**Comment by keybreak** [comment]
@thestinger 
I see your point, it's valid perspective.
And there's certainly no way to have a silver bullet in terms **some** street cops might be not idiots.

However, i also believe that in some countries most people given choice - would gladly take the risk of hiding and playing dumb in such situation, rather than run or show them real data - because then you will have **zero** chance.

I believe having such an option of stealth profile, assuming some day it would be possible technically, would in fact greatly benefit those who want to reduce harm, but it should be clearly stated that you as OS creators have no responsibility on how it is used and people shouldn't blindly rely on illusion of bulletproof solutions, or as you've put it a "theater" (i actually believe you already state it somewhere on a website, if i recall correctly).

**Comment by keybreak** [comment]
Yes, i've read all your posts about Cellebrite...That's pretty scary and complex stuff, especially for countries i were referring to, where law or rights is not a thing.

As far as i understand, currently only thing you can do is refuse to unlock, once device is captured by those who wish to use Cellebrite on you...which means that refusal to unlock - automatically puts you in danger.
Therefore, overall, in terms of physical safety currently it's always a game of personal decision / chances :slightly_frowning_face: 

Oh, unless you know for a fact beforehand that your adversary will use Cellebrite on you, and will be fast enough to use duress password before they'll get their hands on it. I imagine usually there's not enough time to be fast enough, but it's always great idea to have duress password setup just in case.
